src/tests/functional/shared_test_classes/include/shared_test_classes/single_op/softmax.hpp

```diff
@@ -4,12 +4,10 @@
 
 #pragma once
 
-#include "ov_models/builders.hpp"
-
 #include "common_test_utils/common_utils.hpp"
-
 #include "openvino/op/result.hpp"
 #include "openvino/op/softmax.hpp"
+#include "ov_models/builders.hpp"
 #include "shared_test_classes/base/ov_subgraph.hpp"
 
 namespace ov {
```

Comment:
Could we put `SoftMaxLayerTestBase` into `ov::test `namespace?

Reply:
Yes. `aux` namespace is also looks obsolete

Reply:
will put it in the next PR